CMakeLists.txt

```diff
@@ -85,12 +86,12 @@ target_include_directories(qserl
     ${CMAKE_CURRENT_SOURCE_DIR}/src
   )
 
-target_compile_features(qserl PRIVATE cxx_std_11)
+#target_compile_features(qserl PRIVATE cxx_std_11)
 target_compile_options(qserl PRIVATE -Wall -Wextra)
 
 target_link_libraries(qserl
   PUBLIC
-    Boost::boost
+  ${Boost_LIBRARIES}
```

Comment:
I would prefer keeping the use of targets:
`Boost::boost
Boost::python`

Reply:
Done